docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,537 @@
+
+### Debt Management
+This responsibility is taken by callers with DEBT_MANAGER role
+
+This role can increase or decrease strategies specific debt.
+
+The vault sends and receives funds to/from strategies. The function updateDebt(strategy, target_debt) will set the current_debt of the strategy to target_debt (if possible)
+
+If the strategy currently has less debt than the target_debt, the vault will send funds to it.
+
+The vault checks that the `minimumTotalIdle` parameter is respected (i.e. there's at least a certain amount of funds in the vault).
+
+If the strategy has more debt than the max_debt, the vault will request the funds back. These funds may be locked in the strategy, which will result in the strategy returning less funds than requested by the vault. 
+
+#### Setting maximum debt for a specific strategy
+The MAX_DEBT_MANAGER can set the maximum amount of tokens the vault will allow a strategy to owe at any moment in time.
+
+Stored in strategies[strategy].max_debt
+
+When a debt re-balance is triggered, the Vault will cap the new target debt to this number (max_debt)
+
+#### Setting the deposit limit
+The DEPOSIT_LIMIT_MANAGER is in charge of setting the deposit_limit for the vault
+
+On deployment deposit_limit defaults to 0 and will need to be increased to make the vault functional
+
+#### Setting minimum idle funds
+The MINIMUM_IDLE_MANAGER can specify how many funds the vault should try to have reserved to serve withdrawal requests
+
+These funds will remain in the vault unless requested by a Depositor
+
+It is recommended that if no queue_manager is set some amount of funds should remain idle to service withdrawals
+
+#### Setting the profit unlock period
+The PROFIT_UNLOCK_MANAGER is in charge of updating and setting the profit_max_unlock_time which controls how fast profits will unlock
+
+This can be customized based on the vault based on aspects such as number of strategies, TVL, expected returns etc.
+
+#### Setting the default queue
+The QUEUE_MANAGER has the option to set a custom default_queue if desired. The vault will arrange the default queue automatically based only on the order that strategies were added to the vault. If a different order is desired the queue manager role can set a custom queue.
+
+All strategies in the default queue must have been previously added to the vault.
+
+#### Buying Debt
+The DEBT_PURCHASER role can buy debt from the vault in return for the equal amount of `asset`.
+
+This should only ever be used in the case where governance wants to purchase a set amount of bade debt from the vault in order to not report a loss.
+
+It still relies on convertToShares() so will only be viable if the conversion does not reflect and large negative realized loss from the strategy.
+
+
+#### Shutting down the vault
+In an emergency the EMERGENCY_MANAGER can shutdown the vault
+
+This will also give the EMERGENCY_MANAGER the DEBT_MANAGER roles as well so funds can start to be returned from the strategies
```

Comment:
```suggestion
This will also give the `EMERGENCY_MANAGER` the `DEBT_MANAGER` roles as well so funds can start to be returned from the strategies
```